src/main/java/org/folio/list/services/CustomTenantService.java

```diff
@@ -37,7 +42,17 @@ protected void afterTenantUpdate(TenantAttributes tenantAttributes) {
     log.info("Initializing system user");
     prepareSystemUserService.setupSystemUser();
 
-    log.info("Verifying lists are up to date");
-    migrationService.verifyListsAreUpToDate();
+    // In Eureka, the system user often takes a short bit of time for its permissions to be assigned, so retry in the
+    // case of failures related to missing permissions
+    RetryTemplate.builder()
+      .exponentialBackoff(Duration.of(2, ChronoUnit.SECONDS), 1.5, Duration.of(1, ChronoUnit.MINUTES))
```

Comment:
This is a bit different from what we discussed. This will make it retry forever, with the interval between retries starting at 2 seconds, then incrementing with a multiplier of 1.5, up to a max of 1 minute. The `withTimeout()` call below, then sets how long it should do this, so it should eventually give up after 2 minutes. That's like 9 or 10 tries before it gives up

Reply:
I'm good with this as long as the comment above comes true